docs/products/m3db/index.rst

```diff
@@ -33,12 +35,43 @@ Check out our :doc:`getting-started` guide for Aiven for M3 and take your first
     💻 :doc:`howto`
 
 
-M3DB resources
---------------
+Ways to use Aiven for M3
```

Comment:
This section seems out of place in developer documentation, let's focus on technical capabilities and integrations, but the use cases work better on the marketing site (where we hopefully already have this information)

Reply:
I'd like to leave it, and see how it works, this is actually not on the marketing pages, and I am not sure if it's easily understood what people can do with these products.

Reply:
If the goal of this section is to explain what people can do, that does seem useful, but it currently misses that goal and instead strays into marketing-to-developers territory which we would want to avoid. Please refactor the language to give the "When to choose M3" information in terms of technical problems.

Reply:
I would agree with @lornajane, as of now the section seems too much marketing oriented (e.g. "Process **self-driving car data** with faster data ingest and queries, and with stronger data compression."). 

what about "What can you do with M3" section rephrasing the points contained in the section like
* Monitor IoT deployments, applications performance, financial trends and even self-driving cars
* Detect problems, respond promptly to incidents and plan maintenance
* Provide fast data ingest and queries, with strong data compression
* Build statistical models on the historic data to predict future trends

It contains more or less the same info, but with the eye of the developer

Reply:
Thanks both, and thanks @ftisiot for pointing me to the right wording (perspective).

Reply:
Could you swap this bit with the section below about the integrations, and remove the reference to self-driving cars, please. That is a fun example that we could use in a tutorial but not a mainstream or technical application, and the marketing speak undermines the trustworthiness of documentation.

I'm also not sure that we can build statistical models to predict the future with M3 ... @ftisiot would we integrate with other tools for predictions, is that what you had in mind? Let's avoid over-promising in developer documentation if that feature doesn't exist unless we already have a howto on achieving it with an integration.